src/mono/sample/wasm/browser-threads-minimal/main.js

```diff
@@ -33,6 +31,7 @@ try {
     await exports.Sample.Test.DisposeTest();
     console.log("smoke: DisposeTest done ");
 
+    /* FIXME
```

Comment:
might be good to explain why this is commented out

Reply:
I will revert that change. Possibly the sample will fail because the cleanup is not implemented yet. I'm doing it in https://github.com/dotnet/runtime/pull/88052